scripts/ci.sh

```diff
@@ -27,14 +27,16 @@ run_unit_test() {
     echo "Running unit tests with coverage'"
     ./scripts/coverage.sh --coveralls
   else
-    echo "Running unit tests'"
+    echo "Running 'unit tests'"
     make test-unit
   fi
 }
 
 run_style_check() {
   echo "Running 'make test-style'"
   make test-style
+  echo "Running 'make verify-docs'"
+  make verify-docs
```

Comment:
When I tried to run this one a 2) node, it didn't run at all -- I only see a 0 and 1 nodes... that's why I added it here. But maybe I'm not doing it right? I don't have a ton of experience with CircleCI.